client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -345,6 +347,39 @@ export default React.createClass( {
 				section="domain-management" />;
 	},
 
+	pendingTransfer() {
+		const domains = this.getDomains().filter( domain => domain.pendingTransfer );
+
+		if ( domains.length !== 1 ) {
+			return null;
+		}
+
+		const domain = domains[ 0 ];
+
+		const compactNotice = this.translate( '%(domain)s is pending transfer.', {
+				args: {
+					domain: domain.name
+				}
+			} ),
+			fullNotice = this.translate( '%(domain)s is pending transfer. You must wait for the transfer to finish, and then update the settings at the new registrar.', {
+				args: {
+					domain: domain.name
+				}
+			} );
+
+		return (
+			<Notice
+				isCompact={ this.props.isCompact }
+				status="is-error"
```

Comment:
Isn't `error` a bit too much? Warning at most, maybe even just info or default should suffice.